docs/metrics/nli-classification.md

```diff
@@ -0,0 +1,57 @@
+# NLI Classification
+
+The [Cross-Encoder for Natural Language Inference](https://huggingface.co/cross-encoder/nli-deberta-v3-base) (NLI) is an
+text classification model that takes a pair of text and assigns a label: `'contradiction'`, `'entailment'` or
+`'neutral'`. This is useful for hallucination detection, as factual consistency implies the absence of contradictions.
+For any label, the higher the score, the more confident the model is to assign that label. So, it chooses to assign the
+respective label having the highest score.
+
+## Implementation Details
+
+This cross-encoder for NLI classification is based on Microsoft's
+[deberta-v3-base](https://huggingface.co/microsoft/deberta-v3-base), which was trained on NLI data, using
+[`sentence_transformers.cross_encoder`](https://www.sbert.net/docs/package_reference/cross_encoder.html).
+
+Further details for development and usage can be found on Hugging Face:
+[cross-encoder/nli-deberta-v3-base](https://huggingface.co/vectara/hallucination_evaluation_model).
+Below is a quick example of how it can be used:
+
```

Comment:
Should we have a callout here that walks through the installation steps?

Reply:
Adding the necessary pip install, but nothing more. Don't want to go into torch things there